preprocessors/openstreetmap/config.py

```diff
@@ -8,3 +8,10 @@
 
 secondaryServer1 = "https://maps.mail.ru/osm/tools/overpass/api/interpreter"
 secondaryServer2 = "https://overpass-api.de/api/interpreter"
+
+
+# Set names for map servers
+
+map_server_1 = "Pegasus Server"
```

Comment:
I really don't understand the benefit of using these names when we would need to look here to find out the actual URL anyway.

Reply:
@JRegimbal, do you have any suggestions in mind on this?

Reply:
I'd just directly use URLs in log messages.

Reply:
Okay, that has been done now.